common/Services/LocalSettingsService.cs

```diff
@@ -50,6 +51,19 @@ private async Task InitializeAsync()
         }
     }
 
+    public async Task<string[]> EnumerateSettings()

```

Comment:
Who's calling this?

Reply:
will remove - I had that as part of a prior design